crates/node/builder/src/launch/engine.rs

```diff
@@ -258,6 +260,41 @@ where
 
         info!(target: "reth::cli", "Consensus engine initialized");
 
+        // install ress subprotocol if enabled
+        // TODO: where should we keep the sender?
+        let (tx, mut _rx) = tokio::sync::mpsc::unbounded_channel();
```

Comment:
@mattsse pls advise where to keep this sender. no reason for it to be used from reth side, but can't drop either